### DISCUSSION TITLE
Suggestions to streamline sample loading

### DISCUSSION THREAD

**Discussion by davidgritter** [GrandOrgue/GoOdf#98] [Ideas]
Quite often I load samples where the tremmed samples are in an entirely different directory.  When loading the Tremmed samples they are correctly marked as "play when wave trem is on".  However there doesn't appear to be a similar mechanism to load the untremmed samples correctly marked as  "play when wave trem is off"  Perhaps a button or checkbox, at least in the flexible pipe loading popup to load the samples and associated releases with  wave trem off could be added to save a lot of subsequent mouse activity when editing the individual samples and releases.

**Comment by larspalo** [comment]
Have you tried setting the IsTremulant option for one of the non-tremmed attacks with the Ctrl-E to edit it and subsequently use the button "Copy properties to other attacks coming from the same directory"?

**Reply by davidgritter** [reply]
Yes this is what I normally do.  But consider, if working on say 40 stops, with three perspectives each, this amounts to 120 ranks.  For each rank there are at least four instances where this editing is done (1 attack and three releases) each instance requires  three mouse actions, and the one selecting the tremulant setting from the pulldown is very error prone.  Thus there would be a total of 1440 mouse actions that could be eliminated by by having a persistent checkbox or separate button for loading the original samples.

**Reply by larspalo** [reply]
Right, an option checkbox for that could very well be put up on the TODO list of new features to implement.

**Comment by larspalo** [comment]
@davidgritter An option to load pipes with "play when wave tremulant is off" is added in version 0.14.0. That checkbox affects all types of loading except when loading tremulant samples.

**Reply by davidgritter** [reply]
thank you, I see the checkbox and will try it soon.

**Reply by davidgritter** [reply]
I have tried this and it works well,  thank you for the effort.
